src/pages/login.js

```diff
@@ -0,0 +1,85 @@
+import { makeStyles } from "@material-ui/core/styles";
+import { getProviders, getSession } from "next-auth/react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Login from "@/promisetracker/components/LoginPage";
+import Page from "@/promisetracker/components/Page";
+import backendFn from "@/promisetracker/lib/backend";
+import i18n from "@/promisetracker/lib/i18n";
+import wp from "@/promisetracker/lib/wp";
+
+const useStyles = makeStyles(({ breakpoints, typography, widths }) => ({
+  section: {
+    padding: `0 ${typography.pxToRem(23)}`,
+    margin: 0,
+    width: "100%",
+    [breakpoints.up("lg")]: {
+      padding: 0,
+      margin: "0 auto",
+      width: typography.pxToRem(widths.values.lg),
+    },
+  },
+  footer: {
+    marginTop: 0,
+  },
+}));
+
+function SignIn({ providers: providersProp, ...props }) {
```

Comment:
```suggestion
function Login({ providers: providersProp, ...props }) {
```
Or why `/login` then `SignIn`?

Reply:
> Lets add a place to show errors redirected from [here](https://github.com/CodeForAfrica/PromiseTracker/blob/9b16bccebdc7ee5d5c4d3671844b7fe5b291301d/src/pages/api/auth/%5B...nextauth%5D.js#L50)


Was thinking of a 500 page.
If we are reusing this page, then we may not need the 500 page?